cosmic_ray/commands/report.py

```diff
@@ -1,15 +1,19 @@
 def _print_item(item):
-    return [
-        'job ID: {}'.format(item.work_id),
-        'module: {}'.format(item.module_name),
-        'operator: {}'.format(item.operator_name),
-        'occurrence: {}'.format(item.occurrence),
+    result = item.result_data
+    result_type = item.result_type
+    if result_type in ['normal', 'exception']:
+        result_type = result[1][0]
+    ret_val = [
+        'job ID {}:{}:{}'.format(
+                                item.work_id,
+                                result_type,
+                                item.module_name),
```

Comment:
Use PEP8 for this indentation, i.e. indent them 4 spaces in from the previous line. I'm not religious about this, but it lights my editor up when PEP8 is violated and I'll end up changing the lines myself at some point if you don't.